src/io/flutter/module/settings/ProjectType.java

```diff
@@ -95,11 +110,52 @@ public ComboBox getProjectTypeCombo() {
     return projectTypeCombo;
   }
 
-  public void setSdk(@NotNull FlutterSdk sdk) {
-    this.sdk = sdk;
+  public void setSdk(@NotNull Supplier<? extends FlutterSdk> sdk) {
+    this.getSdk = sdk;
+    adjustAndroidX();
```

Comment:
nit: rename from adjustAndroidX to computeAndroidXAvailability() or something.

Reply:
Done.